discovery.md

```diff
@@ -541,16 +541,28 @@ This MUST return an array of zero or more Services. The array MUST contain the
 latest version of all Services available via this Discovery Enpoint.
 
 The collection of services MAY be filtered by supplying one or more
-[service attributes](#service-attributes) as a query parameter.  Doing so MUST
-cause only Services with at least an exact value match for those filters to be
-included in the result.  Parameter names and values are case sensitive.
+[service attributes](#service-attributes) under the `filter` query parameter.
+The general format for the `filter` query parameter is:
 
-Discovery endpoints MUST support filtering with the following attributes and
-MUST reject any unsupported filters.  Endpoints MAY add additional query
-parameters or filter processing to improve upon this minimum provision.
+```
+?filter=ATTRIBUTE=VALUE[,ATTRIBUTE=VALUE]
```

Comment:
Open question: how do people specify a comma in a value? Should we support an escape, or should we not allow multiple per "filter" and require multiple "filter" query params instead? I'm leaning toward forcing multiple query parameters.